File: app/src/main/java/edu/schoolapp/PrimaryMenuType.java
package edu.schoolapp;

public enum PrimaryMenuType {
    TODO {
        @Override
        public String toString() {
            return "Todo";
        }
    }, ASSIGNMENT {
        @Override
        public String toString() {
            return "Assignment";
        }
    }, WEBLINK {
        @Override
        public String toString() {
            return "Web Link";
        }
    }, MAIL {
        @Override
        public String toString() {
            return "Mail";
        }
    }, DRIVE {
        @Override
        public String toString() {
            return "Drive";
        }
    }, TIMETABLE {
        @Override
        public String toString() {
            return "Timetable";
        }
    }, CALENDAR {
        @Override
        public String toString() {
            return "Calendar";
        }
    }
}

File: app/src/main/java/edu/schoolapp/PrimaryMenuType.kt
package edu.schoolapp

enum class PrimaryMenuType {
    TODO {
        override fun toString(): String {
            return "Todo"
        }
    },
    ASSIGNMENT {
        override fun toString(): String {
            return "Assignment"
        }
    },
    WEBLINK {
        override fun toString(): String {
            return "Web Link"
        }
    },
    MAIL {
        override fun toString(): String {
            return "Mail"
        }
    },
    DRIVE {
        override fun toString(): String {
            return "Drive"
        }
    },
    TIMETABLE {
        override fun toString(): String {
            return "Timetable"
        }
    },
    CALENDAR {
        override fun toString(): String {
            return "Calendar"
        }
    }
}